### DISCUSSION TITLE
is it possible to deploy django project on vercel ?

### DISCUSSION THREAD

**Discussion by ilia4bit** [vercel/vercel#6621] [Help] [answered]
hi i'm starting project which use django and django rest framework for backends (just for serving api) not for template rendering
and next.js and react for my frontend is it possible i deploy my django api

**Comment by leerob** [comment, marked as answer]
Hey! I would recommend deploying your Django API on a server platform like Heroku 👍 

https://devcenter.heroku.com/articles/deploying-python

**Reply by ilia4bit** [reply]
thanks for your help so it is ok to deploy projects seperate ? (sorry about this question in deployment topic i'm real noob)

**Reply by leerob** [reply]
Yes! Quite a few teams deploy their frontend on Vercel and their existing API is elsewhere (like Heroku!)

**Reply by ilia4bit** [reply]
thanks for your helpful answer 😄👍🏼
